README.md

```diff
@@ -1,39 +1,149 @@
+
+![First widget example](gifs/widget1.gif)
+</details>
+
+<details><summary>Second widget variation</summary>
+    
+![Second widget example](gifs/widget2.gif)
+</details>
+
+<details><summary>Third widget variation</summary>
+
+![Third widget example](gifs/widget3.gif)
+</details>
+
+<details><summary>Fourth widget variation</summary>
+
+![Fourth widget example](gifs/widget4.gif)
+</details>
 
 ## Features
 
-TODO: List what your package can do. Maybe include images, gifs, or videos.
+By installing this widget in your app, you will help to end the war in Ukraine.
+It will provide links for:
+
+* [donations](https://uahelp.monobank.ua/),
+* sharing this widget
+* and [other ways to support Ukraine](https://war.ukraine.ua/).
 
 ## Getting started
 
-TODO: List prerequisites and provide or point to information on how to
-start using the package.
+Add the package as a dependency to your __pubspec.yaml__:
+
+``` yaml
+dependencies:
+    help_ukraine_widget: <latest version>
+```
+
+in your code:
+
+``` dart
+import 'package:help_ukraine_widget/help_ukraine_widget.dart';
+```
 
 ## Usage
 
-TODO: Include short and useful examples for package users. Add longer examples
-to `/example` folder. 
+### Basic usage
+
+Use `OverlayWidget` to float and align the widget above your app:
+
+``` dart
+OverlayWidget(
+    alignment: Alignment.bottomRight,
+    overlayWidget: HorizontalHelpWidget(),
+    child: MyApp(),
+)
+```
+
+### Customization
+
+You can also create custom variations of this widget:
 
-```dart
-const like = 'sample';
+``` dart
+// we used that in our widget, but you do you 
+final order = [HelpWidgetView.collapsed, HelpWidgetView.main, HelpWidgetView.options];
+
+final _controller = TraverseController(order);
+
+HelpWidget(
+    controller: _controller;
+    // view with links to resources that help Ukraine
+    optionsView: ...
+    // The default first view that the users see.
+    mainView: ...
+    // smallest view that doesn't obstruct user from
+    // using the app, but still exists on the screen
+    collapsedView: ...
+    // axis of transition animation between views
+    axis: ...
+)
 ```
 
-## Additional information
+To make it easier we provide some classes:
+
+#### Components
+
+##### Buttons
+
+`DetailsButton` - used in main view to go to options view.
+
+`HelpOptionButton` - used in options view as a link to external resources.
+
+`RoundedButton` - close button for `CardRounded`.
+
+##### Other
+
+`CardRounded` - base widget for every view.
+
+`ChevronDown` - flexible chevron icon.
+
+`FlagCard` - default collapsed view.
+
+`FlagWidget` - flexible widget with Ukraine flag colors.
+
+`HoverWrapper` - add responses for pointer events to your widget.
+
+`XMark` - flexible cross icon.
+
+### Controllers
+
+`TraverseController` is used to track current view and direction of last transition.
+
+### Helpers
+
+`defaultOptionsList` - list of default link buttons for external resources.
```

Comment:
Let's probably scrap this.

High chance to get this out of sync with real implementations. Much more reliable to redirect to autogenerated API docs as soon as we're out on pub.dev.